### PR TITLE
Fix package warnings about srcdir and  $(cygwin -m /)  Sometimes, min…

### PR DESCRIPTION
**Pull request by JPeterMugaas**
…or updates.

gmime - 3.02
gtksourceview3 - 3.24.4

### CODE REVIEW

File: mingw-w64-gtksourceview3/PKGBUILD [hunk 1]
```diff
@@ -1,10 +1,10 @@
 # Maintainer: Alexey Pavlov <alexpux@gmail.com>
 
 _realname=gtksourceview
-pkgbase=mingw-w64-${_realname}3
+pkgbase=mingw-w64-${_realname}
```
Comment (lazka) [inline mingw-w64-gtksourceview3/PKGBUILD]: Why?

File: mingw-w64-gtksourceview3/PKGBUILD [hunk 2]
```diff
@@ -14,14 +14,14 @@ makedepends=("${MINGW_PACKAGE_PREFIX}-gcc"
              "${MINGW_PACKAGE_PREFIX}-gobject-introspection"
              "${MINGW_PACKAGE_PREFIX}-glade"
              "${MINGW_PACKAGE_PREFIX}-vala"
-             "${MINGW_PACKAGE_PREFIX}-gtk-doc"
-             "intltool")
+             "intltool"
+             "gtk-doc")
```
Comment (lazka) [inline mingw-w64-gtksourceview3/PKGBUILD]: Why the change back to gtk-doc? Doesn't it build or you?

File: mingw-w64-gtkspell3/PKGBUILD [hunk 3]
```diff
@@ -4,7 +4,7 @@ _realname=gtkspell3
 pkgbase=mingw-w64-${_realname}
 pkgname="${MINGW_PACKAGE_PREFIX}-${_realname}"
 pkgver=3.0.9
-pkgrel=2
+pkgrel=1
```
Comment (lazka) [inline mingw-w64-gtkspell3/PKGBUILD]: Why decrease pkgrel?

**Comment by Alexpux** [comment]
@JPeterMugaas what about other comments?

File: mingw-w64-gtksourceview3/PKGBUILD (hunk 2 above)
Comment (JPeterMugaas) [reply mingw-w64-gtksourceview3/PKGBUILD]: I have taken the liberty of removing the "gtk-doc" reference and added 3 to pkgbase.

File: mingw-w64-glib2/PKGBUILD [hunk 4]
```diff
@@ -7,7 +7,7 @@ _realname=glib2
 pkgbase=mingw-w64-${_realname}
 pkgname="${MINGW_PACKAGE_PREFIX}-${_realname}"
 pkgver=2.54.0
-pkgrel=2
+pkgrel=3
```
Comment (Alexpux) [inline mingw-w64-glib2/PKGBUILD]: Why need rebuild glib2?

File: mingw-w64-gtksourceview3/PKGBUILD [hunk 5]
```diff
@@ -44,8 +43,7 @@ build() {
     --host=${MINGW_CHOST} \
     --enable-shared \
     --disable-static \
-    --enable-glade-catalog \
-    --enable-gtk-doc
```
Comment (Alexpux) [inline mingw-w64-gtksourceview3/PKGBUILD]: Why disable gtk-doc?

File: mingw-w64-gtkspell3/PKGBUILD [hunk 6]
```diff
@@ -14,7 +14,7 @@ makedepends=("${MINGW_PACKAGE_PREFIX}-gcc"
              "${MINGW_PACKAGE_PREFIX}-iso-codes"
              "${MINGW_PACKAGE_PREFIX}-pkg-config"
              "${MINGW_PACKAGE_PREFIX}-vala"
-             "${MINGW_PACKAGE_PREFIX}-gtk-doc"
+             "gtk-doc"
```
Comment (Alexpux) [inline mingw-w64-gtkspell3/PKGBUILD]: Return mingw-gtk-doc

File: mingw-w64-gtkspell3/PKGBUILD [hunk 7]
```diff
@@ -41,8 +41,7 @@ build() {
     --host=${MINGW_CHOST} \
     --enable-shared \
     --disable-static \
-    --enable-gtk2 \
-    --enable-gtk-doc
```
Comment (Alexpux) [inline mingw-w64-gtkspell3/PKGBUILD]: Leave enable-gtk-doc

File: mingw-w64-libpeas/PKGBUILD [hunk 8]
```diff
@@ -4,18 +4,18 @@ _realname=libpeas
...
+url="https://www.gtk.org"
 license=("LGPL2")
 makedepends=("${MINGW_PACKAGE_PREFIX}-gcc"
              "${MINGW_PACKAGE_PREFIX}-glade"
              "${MINGW_PACKAGE_PREFIX}-gobject-introspection"
              "${MINGW_PACKAGE_PREFIX}-pkg-config"
              "${MINGW_PACKAGE_PREFIX}-python3-gobject"
              "${MINGW_PACKAGE_PREFIX}-python2-gobject"
-             "${MINGW_PACKAGE_PREFIX}-gtk-doc"
+             "gtk-doc"
```
Comment (Alexpux) [inline mingw-w64-libpeas/PKGBUILD]: The same as above

**Comment by JPeterMugaas** [comment]
I have tried to fix some of the problems and added an update to dbus.

**Comment by lazka** [comment]
Mostly out of date now, sorry :/
